Tests/Engine/Settings.tests.ps1

```diff
@@ -91,27 +91,27 @@ Describe "Settings Class" {
         }
 
         It "Should return 2 IncludeRules" {
```

Comment:
hmmm, this doesn't seem right, I assume the number is right so the title of these should probably change

Reply:
I agree. I will make it a variable to future proof.